apps/frontend/src/libs/hooks/use-intersection-observer/use-intersection-observer.hook.ts

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useRef } from "react";
+
+const FIRST_ITEM = 0;
```

Comment:
Maybe it's better to put it in a separate file

Reply:
I thought about it too, but I don't see any reason for it. We already used a similar local constant in the avatar component, so I decided to do the same.
```
const FIRST_LETTER_INDEX = 0;

const Avatar = ({ imageUrl, name }: Properties): JSX.Element => {
	const firstLetter = name[FIRST_LETTER_INDEX];
	const hasImage = Boolean(imageUrl);
```